CONTRIBUTING.md

```diff
@@ -103,16 +103,23 @@
       - Then run `yarn local`
    5. Please include `test` user locally to test queries in your database.
       ```
-      CREATE DATABASE test;
-      CREATE ROLE postgres WITH LOGIN NOSUPERUSER INHERIT CREATEDB NOCREATEROLE NOREPLICATION PASSWORD 'test!';
-      GRANT CONNECT ON DATABASE test TO postgres;
-      GRANT ALL PRIVILEGES ON DATABASE test TO postgres;
-      // Inside `test` database create `test1` schema.
-      CREATE SCHEMA test1;
+      $ psql postgres
+      postgres=# CREATE ROLE tester WITH LOGIN NOSUPERUSER INHERIT CREATEDB NOCREATEROLE NOREPLICATION PASSWORD 'test!';
+      postgres=# GRANT ALL PRIVILEGES ON DATABASE test TO tester;
+
+      postgres=# exit
+
+      // Connect psql with user newly created 'tester'
+      $ psql postgres -U tester
+      // create 'test' database to be owner
+      postgres=> CREATE DATABASE test;
+      // Connect Inside `test` database to create `test` schema.
+      postgres=> \connect test
+      test=> CREATE SCHEMA test;
       ```
       - Above should match `test.env`
         ```
-        DATABASE_URL="postgresql://test:test!@localhost:5432/postgres?schema=test"
+        DATABASE_URL="postgresql://tester:test!@localhost:5432/test?schema=test"
```

Comment:
Not a good idea since this will be included in `git`. Should be better to match `DATABASE_URL` already provided.